fix(image): introduce a new setting `start-pre-pull-manager-images` (#922)
longhorn/website: content/docs/1.7.0/references/helm-values.md

| defaultSettings.snapshotDataIntegrityCronjob | Setting that defines when Longhorn checks the integrity of data in snapshot disk files. You must use the Unix cron expression format. |
| defaultSettings.snapshotDataIntegrityImmediateCheckAfterSnapshotCreation | Setting that allows disabling of snapshot hashing after snapshot creation to minimize impact on system performance. |
| defaultSettings.snapshotMaxCount | Maximum snapshot count for a volume. The value should be between 2 to 250 |

> | defaultSettings.startPrePullManagerImages | Pre-pull Longhorn manager images on all worker nodes before starting the manager pods. |

Reviewer: ```suggestion
| defaultSettings.startPrePullManagerImages | Setting that allows worker nodes to pull Longhorn Manager images before Longhorn Manager pods are started. |
```
